beacon_node/execution_layer/src/engine_api.rs

```diff
@@ -197,6 +203,10 @@ pub struct ExecutionBlockWithTransactions<E: EthSpec> {
     #[superstruct(only(Deneb, Electra))]
     #[serde(with = "serde_utils::u64_hex_be")]
     pub excess_blob_gas: u64,
+    #[superstruct(only(Electra))]
+    pub deposit_receipts: Vec<JsonDepositRequest>,
```

Comment:
This can just be named `DepositRequest` as the consensus data structure is `DepositReceipt`. Or if we need to namespace call it `EngineDepositRequest` or similar